agent/logger/audit/entry_types.go

```diff
@@ -38,7 +40,11 @@ const (
 	// 8. version
 	// 9. cluster
 	// 10. container instance arn
-	getCredentialsAuditLogVersion = 1
+
+	// Version '2', following fields were modified
+	// 1. event type ('GetCredentials, GetCredentialsExecutionRole')
```

Comment:
would be better use `7` here, instead of `1`?

Reply:
that was my initial thought too, but also thought it might be confusing. i can change it, i dont feel strongly either way.